### PR TITLE
[spec] version_spec: forget about rev and commit

### PR DESCRIPTION
**Pull request by Frenzie**
Stable doesn't have them.

### CODE REVIEW

File: spec/unit/version_spec.lua [hunk 1]
```diff
@@ -6,16 +6,12 @@ describe("Version module", function()
     end)
     it("should get current revision", function()
         local rev = Version:getCurrentRevision()
-        local year, month, revision = rev:match("v(%d%d%d%d)%.(%d%d)-?(%d*)")
-        local commit = rev:match("-%d*-g(%x*)[%d_%-]*")
+        local year, month, point, revision = rev:match("v(%d%d%d%d)%.(%d%d).(%d?)-?(%d*)") -- luacheck: ignore 211
```
Comment (Frenzie) [inline spec/unit/version_spec.lua]: @NiLuJe On the flipside, it's probably just the `.1` that was tripping it up.

Comment (Frenzie) [reply spec/unit/version_spec.lua]: I mean, in a PR here on GH it'll presumably always be v2018.11.1-1-gasdafsdf at the very lowest, right?

**Comment by Frenzie** [comment]
Quite so! But of course, with the way the test is written right now the whole backside of that pattern might just as well be chopped off. I didn't quite want to waste the effort that went into writing it though. (It always takes me a short while to wrap my head around Lua patterns because it looks so much like regex.)

**Comment by NiLuJe** [comment]
Yeah, I always end up having to fire up a luajit shell to double-check ;).

**Comment by Frenzie** [comment]
I'll merge it to unbreak the unit tests.

**Pull request merged**